src/globals.js

```diff
@@ -7,4 +7,14 @@ export const clearTimeout = global.clearTimeout;
 
 export const document = window && window.document;
 export const navigator = window && window.navigator;
-export const sessionStorage = window && window.sessionStorage;
+
+export const sessionStorage = ( function() {
```

Comment:
this `const sessionStorage` shadows the global one, forcing the `sessionStorage` in the function scope (line 14) to always be undefined.